### PR TITLE
[coverage] More tests for inet.py

### PR DESCRIPTION
**Pull request by guedou**
This PR test the `report_ports()` function as well as the `TCP_client()`  automaton. A new `linux.uts` was to tests Linux specific parts of Scapy. Here, it is used to call iptables in order to test the TCP automaton.

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ test_script:
   - 'del test\regression.uts'
 
   # Secondary and contrib unit tests
-  - 'del test\bpf.uts' # Don't bother with BPF regression tests
+  - 'del test\bpf.uts linux.uts' # Don't bother with OS dependent regression tests
```
Comment (gpotter2) [inline .appveyor.yml]: `test\linux.uts` :)

Comment (guedou) [inline .appveyor.yml]: Thanks !

**Comment by guedou** [comment]
@p-l- do you understand why the HomePlugAV tests fail ?

File: test/linux.uts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+IPTABLE_RULE = "iptables -%c INPUT -s %s -p tcp --sport 80 -j DROP"
+
+# Drop packets from SECDEV_IP4
+assert(os.system(IPTABLE_RULE % ('A', SECDEV_IP4)) == 0)
+
+success = False
+for i in xrange(5):
+  r,w = os.pipe()
+  t = TCP_client(SECDEV_IP4, 80, external_fd={ "tcp": (r,w) })
+  _ = os.write(w, "HEAD / HTTP/1.0\r\n\r\n")
```
Comment (gpotter2) [inline test/linux.uts]: Here is your bug. Using `_` as a var breaks it usage :)
```
>>> _ = "wrong"
>>> "test"
'test'
>>> _
'wrong'
>>>
```

**Comment by guedou** [comment]
@gpotter2 thanks for your help !

**Comment by guedou** [comment]
@p-l- this PR is more stable than it used to be and can be merged: the TCP_client automaton test retries on errors and attempts to reload itself on freeze.

**Pull request merged**